gce/image/build_image.sh

```diff
@@ -31,12 +31,14 @@ print_usage() {
     echo "  --download-no-server download all rpms needed excluding scylla using .repo provided in --repo-for-install"
     echo "  --dry-run            validate template only (image is not built)"
     echo "  --build-id           Set unique build ID, will be part of GCE image name"
+    echo "  --log-file           Path for log. Default build/packer.log on current dir"
```

Comment:
I need to parse it by the pipeline.
So I want to be sure that I use the same file, so in case someone will change the default - I still have my value.
I left the default as it, so other users don't need to use it.